self-paced-labs/vertex-ai/vertex-ai-qwikstart/lab_exercise.ipynb

```diff
@@ -1213,68 +1349,74 @@
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "23/23 [==============================] - 0s 1ms/step - loss: 24844828.0000 - mae: 1228.9453 - mse: 24844828.0000 - rmse: 2534.6929\n",
-      "[24844828.0, 1228.9453125, 24844828.0, 2534.69287109375]\n"
+      "Epoch 1/10\n",
+      "  2/164 [..............................] - ETA: 4s - loss: 10593.9062 - mae: 10593.9062 - mse: 735473664.0000 - rmse: 26785.0820WARNING:tensorflow:Callbacks method `on_train_batch_end` is slow compared to the batch time (batch time: 0.0035s vs `on_train_batch_end` time: 0.0532s). Check your callbacks.\n",
+      "WARNING:tensorflow:Callbacks method `on_train_batch_end` is slow compared to the batch time (batch time: 0.0035s vs `on_train_batch_end` time: 0.0532s). Check your callbacks.\n",
+      "164/164 [==============================] - 1s 4ms/step - loss: 1944.2853 - mae: 1944.2853 - mse: 105602640.0000 - rmse: 4977.7876 - val_loss: 1165.3185 - val_mae: 1165.3185 - val_mse: 13002700.0000 - val_rmse: 2366.0540\n",
+      "Epoch 2/10\n",
+      "164/164 [==============================] - 0s 2ms/step - loss: 1642.0222 - mae: 1642.0222 - mse: 96051920.0000 - rmse: 4514.5591 - val_loss: 1085.3497 - val_mae: 1085.3497 - val_mse: 12666133.0000 - val_rmse: 2322.3108\n",
+      "Epoch 3/10\n",
+      "164/164 [==============================] - 0s 2ms/step - loss: 1596.7908 - mae: 1596.7908 - mse: 96331624.0000 - rmse: 4514.3633 - val_loss: 1041.7750 - val_mae: 1041.7750 - val_mse: 13028899.0000 - val_rmse: 2336.2056\n",
+      "Epoch 4/10\n",
+      "164/164 [==============================] - 0s 2ms/step - loss: 1560.9375 - mae: 1560.9375 - mse: 95126792.0000 - rmse: 4462.5308 - val_loss: 1021.7502 - val_mae: 1021.7502 - val_mse: 13083770.0000 - val_rmse: 2327.4673\n",
+      "Epoch 5/10\n",
+      "164/164 [==============================] - 0s 2ms/step - loss: 1542.6259 - mae: 1542.6259 - mse: 94003480.0000 - rmse: 4408.8760 - val_loss: 1018.1912 - val_mae: 1018.1912 - val_mse: 13910818.0000 - val_rmse: 2377.1240\n",
+      "Epoch 6/10\n",
+      "164/164 [==============================] - 0s 2ms/step - loss: 1522.8849 - mae: 1522.8849 - mse: 93518576.0000 - rmse: 4383.8145 - val_loss: 1012.3921 - val_mae: 1012.3921 - val_mse: 13977303.0000 - val_rmse: 2380.1821\n",
+      "Epoch 7/10\n",
+      "164/164 [==============================] - 0s 2ms/step - loss: 1498.6212 - mae: 1498.6212 - mse: 92280416.0000 - rmse: 4310.6851 - val_loss: 1010.1893 - val_mae: 1010.1893 - val_mse: 14495935.0000 - val_rmse: 2406.2073\n",
+      "Epoch 8/10\n",
+      "164/164 [==============================] - 0s 2ms/step - loss: 1490.8345 - mae: 1490.8345 - mse: 91274680.0000 - rmse: 4290.9722 - val_loss: 1011.7308 - val_mae: 1011.7308 - val_mse: 15687941.0000 - val_rmse: 2458.8572\n"
      ]
     }
    ],
    "source": [
-    "print(model.evaluate(evalds))"
+    "history = model.fit(trainds,\n",
+    "                    validation_data=evalds,\n",
```

Comment:
you earlier renamed `evalds` to `devds`, so here evalds not found

Reply:
Done